ui/src/app/zones/views/ZonesListForm/ZonesListForm.tsx

```diff
@@ -21,24 +22,24 @@ const ZonesListForm = ({ closeForm }: Props): JSX.Element => {
   const errors = useSelector(zoneSelectors.errors);
   const saved = useSelector(zoneSelectors.saved);
   const saving = useSelector(zoneSelectors.saving);
-
-  useEffect(() => {
-    dispatch(actions.fetch());
-  }, [dispatch]);
+  const cleanup = useCallback(() => zoneActions.cleanup(), []);
```

Comment:
@sowasred2012 @bethcollins92 @ClementChaumel this cleanup function is important to pass to `FormikForm` because it resets the `saving`, `saved` and `errors` state when the form closes. It's wrapped in a `useCallback` so that the function doesn't get re-instantiated every time the component renders. While not absolutely necessary, it's an easy performance booster.